packages/client/src/lib.js

```diff
@@ -144,10 +189,11 @@ class Web3Storage {
    * const cid = await client.store(car)
    * console.assert(cid === root)
    * ```
-   * @param {Blob} blob
+   * @param {Iterable<API.Web3File>} files
+   * @param {API.PutOptions} [options]
    */
-  store(blob) {
-    return Web3Storage.store(this, blob)
+  put(files, options) {
```

Comment:
How are we going to expose the CID to the user before upload?

Reply:
I am thinking on an option:

`onCar(cid: CID)`, but I am leaving this to follow up PR to discuss